app/components/chat/ImportFolderButton.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import type { Message } from 'ai';
+import { toast } from 'react-toastify';
+
+interface ImportFolderButtonProps {
+  className?: string;
+  importChat?: (description: string, messages: Message[]) => Promise<void>;
+}
+
+const IGNORED_FOLDERS = ['node_modules', '.git', 'dist', 'build', '.next', 'coverage', '.cache', '.vscode', '.idea'];
```

Comment:
You may want to include [ignore](https://www.npmjs.com/package/ignore)

You can check that part in my previous PR #289

Reply:
Good idea, thanks, will add that!

Reply:
Just pushed support for it along with ignoring of binnary files.
I have idea of what to do with binary files but now now. In another PR